### PR TITLE
Add: filter MTK video and image driver in udevadm (BugFix)

### PR DESCRIPTION
**Pull request by baconYao**
## Description
Mediatek contributes some drivers for video and image purpose. However, these drivers cause the Checkbox mistakes them for Camera jobs. In order to avoid this problem, filter those related drivers from udevadm is the way.

Drivers:
  - mtk-mdp3
    - MediaTek image processor 3 driver
  - mtk-jpeg
    - MediaTek JPEG codec driver
  - mtk-vcodec-enc
    - Mediatek video codec V4L2 encoder driver
  - mtk-vcodec-dec
    - Mediatek video codec V4L2 decoder driver

## Resolved issues

Avoid to generate the invalid camera jobs (Not physical camera). The following links are the test result after filtering those drivers from udevadm

- G350
  - Submission: https://certification.canonical.com/hardware/202202-29979/submission/331163/test-results/?term=camera
  - Environment: One physical MIPI ISP camera
  - Expectation: Only camera job for ISP camera

- G700
  - Submission: https://certification.canonical.com/hardware/202308-31931/submission/331160/test-results/?term=camera
  - Environment: No physical camera be connected
  - Expectation: No camera job be generated
 
- G1200
  - Submission: https://certification.canonical.com/hardware/202307-31864/submission/331161/test-results/?term=camera
  - Environment: One physical USB camera
  - Expectation: Only camera job for USB camera

### CODE REVIEW

File: checkbox-support/checkbox_support/parsers/udevadm.py [hunk 1]
```diff
@@ -561,9 +561,14 @@ def category(self):
...
                 return "HIDRAW"
             if self._environment["SUBSYSTEM"] == "video4linux":
-                if self.driver not in ('bcm2835-codec', 'bcm2835-isp'):
-                    # Ignore raspberrypi memory to memory (M2M) devices,
-                    # for the video decoder, encoder, and ISP resize.
+                # Ignore raspberrypi memory to memory (M2M) devices,
+                # for the video decoder, encoder, and ISP resize.
+                # Ignore Mediatek v4l2 encoder, decoder, jpeg codec and
+                # image processor 3
+                driver_black_list = ('bcm2835-codec', 'bcm2835-isp',
```
Comment (pieqq) [inline checkbox-support/checkbox_support/parsers/udevadm.py:568]: Can you use `drivers_blocklist` for this variable instead?

**Review by pieqq — CHANGES_REQUESTED** [review]
Just a tiny spelling change, and this should be good to land! Thanks.

File: checkbox-support/checkbox_support/parsers/udevadm.py (hunk 1 above)
Comment (baconYao) [reply checkbox-support/checkbox_support/parsers/udevadm.py]: Hi @pieqq, I updated it. Please review it again. Thanks

**Review by pieqq — APPROVED** [review]
+1, thanks!

**Review by yphus — APPROVED** [review]
+1

**Comment by yphus** [comment]
@baconYao Hi, we can't merge your branch because commits are not signed, please check how to activate them on your GH account ([doc](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification))

**Comment by baconYao** [comment]
Hi @yphus, I signed it. Please check it again. Many thanks.

**Comment by diohe0311** [comment]
/canonical/self-hosted-runners/run-workflows f8ab6ace12bcf49a9fa92252d486e79fb72c99a9

**Pull request merged**